style.css

```diff
@@ -0,0 +1,147 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@400;700&display=swap');
+
+* {
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+}
+
+body {
+  font-family: 'Poppins', sans-serif;
+  height: 100vh;
+}
+
+#container {
+  padding: 1rem;
+  height: 100%;
+}
+
+#navigation {
+  padding: 0 8rem;
+  display: flex;
+  justify-content: space-between;
+}
+
+#menu {
+  display: flex;
+  justify-content: space-between;
+}
+
+.big {
+  margin-right: 2rem;
+}
+
+#logo {
+  color: #6070ff;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 18px;
+  line-height: 20px;
+}
+
+li {
+  list-style: none;
+}
+
+#hidden {
+  display: none;
+  color: #6070ff;
+}
+
+#bottom {
+  display: flex;
+  align-items: center;
+  background-image: url(img/Header.png);
```

Comment:
- [ ] Since we're doing the mobile version of the portfolio, set the background image to match the design in Figma.

This is from the template: 

![image](https://user-images.githubusercontent.com/79114233/138188207-3c5249be-9577-4a24-90f6-386b4fced460.png)

and this is yours:

![image](https://user-images.githubusercontent.com/79114233/138188243-1d9f23fd-61a1-49de-91cd-36f24ed47a17.png)

In the upcoming milestones, you will work on how to set the image so it matches in both mobile and desktop versions. :tada:

Reply:
Total misunderstanding of the task from my end. Please, see the changes I implemented and for now ignore "Desktop" version. I think the mobile one should be OK.